code-generator/my_mod/util.py

```diff
@@ -100,29 +74,58 @@ def GetCommitHash_(path):
         )
         return result.stdout.strip()
     except subprocess.CalledProcessError:
-        return "0000000000000000000000000000000000000000"
+        print("failed to get commit hash(" + path + ")")
```

Comment:
こっちも標準エラー出力にだしますか？

Reply:
今，code genのエラーメッセージ，全部標準出力なので，一旦全部標準出力にしても良いかもしれないですね（ `GetRepo_` のほうは stderr 出力してくれてるけれども）

あと，今エラーメッセージのフォーマットが
```
Error: ~~
```
なので
```suggestion
        print("Warning: Failed to get commit hash(" + path + ")")
```
としたいです．

Reply:
これエラーじゃなくて warning なんで，stderr にし忘れてただけですね

Reply:
これはシンプルに嘘では > 今，code genのエラーメッセージ，全部標準出力

Reply:
`Warn: ` prefix は付けます

Reply:
> これはシンプルに嘘では > 今，code genのエラーメッセージ，全部標準出力

Reply:
```
print("failed to get commit hash(" + path + ")", file=sys.stderr)
```
もprefixおねがい > util.py L77

Reply:
prefix 足した > util.py L77

Reply:
thx